demos.js

```diff
@@ -761,7 +743,7 @@ const allDemos = [
         "group": "Poo-Brain",
         "youtube": "EIp7ffUrUCs",
         "date": "2017-08-15",
-        "screenshot": "https://content.pouet.net/files/screenshots/00071/00071570.jpg",
+        "screenshot": "https://media.demozoo.org/screens/o/d1/7a/012a.146494.jpg",
```

Comment:
The other screenshot of the same scene could make a more interesting preview:
https://media.demozoo.org/screens/o/22/14/5827.146493.jpg